src/ServiceControl/Recoverability/API/GroupFetcher.cs

```diff
@@ -62,14 +62,19 @@ private static UnacknowledgedRetryOperation[] MapAcksToOpenGroups(FailureGroupVi
                     select unack).ToArray();
         }
 
-        private static FailureGroupView[] GetDBGroups(string classifier, IDocumentSession session)
+        private static FailureGroupView[] GetDBGroups(IDocumentSession session, string classifier, string classifierFilter)
         {
             var groups = session.Query<FailureGroupView, FailureGroupsViewIndex>()
-                .Where(v => v.Type == classifier)
-                .OrderByDescending(x => x.Last)
+                .Where(v => v.Type == classifier);
+
+            if (!string.IsNullOrWhiteSpace(classifierFilter))
+            {
+                groups = groups.Where(v => v.Title == classifierFilter);
```

Comment:
Is equality the only type of filter? Should these parameters be `type` and `title`?

Does case matter here?